src/common/database.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import os
+from enum import Enum
+
+import boto3
+import mysql.connector
+import mysql.connector.pooling
+
+logger = logging.getLogger(__name__)
+
+
+class Table(Enum):
+    UCFS = "ucfs"
+    EQUALITIES = "equalities"
+    AUDIT = "audit"
```

Comment:
I don't think we need this as we aren't targetting specific tables?

Reply:
Removed this snippet as the only usage of the class was removed.